visual-tests/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "playwright",
+  "version": "1.0.0",
```

Comment:
Is this package going to be versioned with each release? 
If so, this process needs to be included in the release script. If not, maybe we could change the version to `0.0.0`, [similarly to `docs`](https://github.com/handsontable/handsontable/blob/12.2.0/docs/package.json#L4) to make it more obvious.

Reply:
`Is this package going to be versioned with each release?`
probably no. Version changed to proposed one.